modules/javafx.controls/src/main/java/javafx/scene/control/SkinBase.java

```diff
@@ -82,8 +83,10 @@
      * want to adjust the way listeners are added rather than continuing to use this
      * map (although it doesn't really do much harm).
      */
+    @Deprecated // replace with listenerHelper
```

Comment:
We only use `@Deprecated` annotation on public API, so you should limit this to a comment.